src/Storefront/Page/LandingPage/LandingPageLoader.php

```diff
@@ -41,19 +38,22 @@ public function load(Request $request, SalesChannelContext $context): LandingPag
         $landingPage = $this->landingPageRoute->load($landingPageId, $request, $context)->getLandingPage();
 
         if ($landingPage->getCmsPage() === null) {
-            throw new PageNotFoundException($landingPageId);
```

Comment:
thank you very much, for going the extra mile :muscle: :+1: 

but unfortunately this would be a break. if someone relies on `PageNotFoundException` being throw, their code will no longer work. 

we need to surround it, with a feature flag condition for 6.8. see here for an example: https://github.com/shopware/shopware/blob/trunk/src/Core/Checkout/Cart/Order/Api/OrderRecalculationController.php#L154-L158